### ISSUE TITLE
Identify the package manager

### ISSUE THREAD

**Issue by pizzamaker** [woodruffw/screenfetch-c#22]
It would be useful to have the package manager identified. Many users take pride in their choice for pkg mgmt :)

**Comment by joedf** [comment]
Cool idea :+1: 
However, I think it is possible to have multiple package manager, am I correct?

**Comment by pizzamaker** [comment]
Yes! Maybe they can be listed alongside each other, with the main one showing up first? The main one could be determined as the manager with the most packages installed on the system, perhaps.

**Comment by woodruffw** [comment]
The package manager is identified when doing package counts, but actually printing out its name isn't done. If you want something like this, please feel free to open a PR (`detect_pkgs` in *src/plat/{linux,bsd,darwin,sun,win32}/detect.c* is a good place to start).

**Comment by pizzamaker** [comment]
I'm not actually a programmer, but I'm glad if anyone else offers to work on it :)

**Comment by cuberobenicio32** [comment]
I'll try to work on it :-)
